pontoon/base/models.py

```diff
@@ -213,6 +219,14 @@ class Meta:
             ('can_manage_locale', 'Can manage locale')
         )
 
+    def get_plural_index(self, cldr_plural):
```

Comment:
Maybe `get_relative_plural_id` to make it more obvious how it's different from `cldr_plural_to_id`?